### ISSUE TITLE
Insufficient information on image conversion failure

### ISSUE THREAD

**Issue by luis100** [keeps/roda#917]
When trying to create a dissemination to JPEG (default parameters) using the file
[balloon_eciRGBv2.tif.zip](https://github.com/keeps/roda/files/952715/balloon_eciRGBv2.tif.zip) (unzip first)

The process fails:
![roda - d146e4d9-c19d-49c6-8ace-2afd33288f68-89b6690c-9585-3ec2-83c3-de2a29505ba1](https://cloud.githubusercontent.com/assets/631728/25352197/14c84c42-2923-11e7-965a-1a3d1c1f89bb.jpg)

Running by the command line:
```
$ /usr/bin/convert -geometry 1600x1600\> -quality 80 -strip -regard-warnings ~/.roda2/data/storage/aip/07c1e843-3167-4209-85f8-dc0361bfdafb/representations/ef20d9f6-45f8-49d5-a3f3-c65415bb7be8/data/images/balloon_eciRGBv2.tif jpg:/home/lfaria/converted902493642290549833.jpg
convert: Incompatible type for "RichTIFFIPTC"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/912.
```

The report should include the information outputted by the command as it is the reason for failure (although just a warning it is indistinguishable from a fatal error).

**Comment by jmaferreira** [comment]
I agree. That should actually be the case for all the plugins.

**Comment by nunovieira220** [comment]
Solved and done to all conversion plugins on a6770a5f80e1a528e837d6e43f63605adce89e53
